docs: apply Snyk writing-style skill to Developer tools pages (#1369)
snyk/user-docs: developer-tools/snyk-cli/commands/monitor.md

### `--maven-skip-wrapper`

> Forces the use of a globally installed `mvn` command, even when a Maven wrapper (that is, `mvnw` or `mvnw.cmd`) is present in the project.

Reviewer: ```suggestion
Forces the use of a globally installed `mvn` command, even when a Maven wrapper (such as `mvnw` or `mvnw.cmd`) is present in the project.
```

### `--dotnet-runtime-resolution`

> **Note:** This option is in Early Access and may change until it is released.

Reviewer: ```suggestion
**Note**: This option is in Early Access and may change until it is released.
```

Required. You must use this option when you test .NET projects using Runtime Resolution Scanning.

Example: `snyk test --dotnet-runtime-resolution`

### `--dotnet-target-framework`

> **Note:** This option is in Early Access and may change until it is released.

Reviewer: ```suggestion
**Note**: This option is in Early Access and may change until it is released.
```
